catalog/chaos-gpt-destruction-plot.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Chaos-GPT, a bot built atop Auto-GPT, planned how it would destroy humanity and posted about it on Twitter
+companies: ["OpenAI"]
+categories: ["Deception", "Aggression"]
+demonstrative: false
+modalities: ["Closed-Loop System"]
```

Comment:
Hmm. I feel like I usually don't hear `Closed-Loop System` used to refer to these agents. Instead, I usually hear `Autonomous System` or `Agent`. Should we classify the modality as an `Agent` or something else instead?